### ISSUE TITLE
Config Change with 0.17.1

### ISSUE THREAD

**Issue by ulrichSchreiner** [sbstp/kubie#114]
hi,

i have 0.17.1 installed and my `kubie.yaml` is no invalid. when doing a `kubie ctx <tab>` i get

~~~
   ~ ❯ kubie ctx …Error: could not parse kubie config                                                                                                                                                                                 ✘ INT

Caused by:
    behavior.print_context_in_exec: unknown variant `auto`, expected one of `Auto`, `Always`, `Never` at line 59 column 28
Error: could not parse kubie config

Caused by:
    behavior.print_context_in_exec: unknown variant `auto`, expected one of `Auto`, `Always`, `Never` at line 59 column 28
Error: could not parse kubie config

Caused by:
    behavior.print_context_in_exec: unknown variant `auto`, expected one of `Auto`, `Always`, `Never` at line 59 column 28
~~~

in the config-file i have to change the line

~~~
    print_context_in_exec: auto
~~~

to

~~~
    print_context_in_exec: Auto
~~~

**Comment by sbstp** [comment]
Sorry about that. A little error when I refactored from structopt to clap. Fixed it and will publish 0.17.2

**Comment by sbstp** [comment]
Should be good now
